test/specs/suite2.spec.js

```diff
@@ -0,0 +1,168 @@
+const { describe, it } = require("mocha");
+
+const loginData = require("../../data/login.json");
+
+const loginPage = require("../../pageObject/loginPage");
+
+const inventoryPage = require("../../pageObject/inventoryPage");
+
+const shoppingCart = require("../../pageObject/shoppingCart");
+const checkOutStep1 = require("../../pageObject/checkOutStep1");
+const checkOutStep2 = require("../../pageObject/checkOutStep2");
+
+describe("WDIO-Home-Task-2-06-18", () => {
+  beforeEach("login and redirection to the inv page", async () => {
+    await loginPage.open();
+    await loginPage.signIn(loginData.valid.userName, loginData.valid.password);
+    await expect(inventoryPage.productTitle).toBeDisplayed();
+  });
+
+  it("should perform and verify high-low sorting on the Inventory page", async () => {
+    const arrayOfTitlesInv = await inventoryPage.arrayOfTitles.sort();
+    
+    const arrayOfPricesInv = await inventoryPage.arrayOfPrices
+    const az = await arrayOfTitlesInv;
+    const za = await arrayOfTitlesInv.reverse();
+    const lohi = await arrayOfPricesInv.sort((a, b) => a - b);
+    const hilo = await arrayOfPricesInv.sort((a, b) => b - a);
+
+    await inventoryPage.sortByOption("hilo").click();
+
+    const itemTitles = await inventoryPage.arrayOfTitles;
```

Comment:
are you using it in this test?

Reply:
no, commented it out - will update to create a function